github/src/main/kotlin/gropius/sync/github/model/IssueInfo.kt

```diff
@@ -11,6 +11,7 @@ import java.time.OffsetDateTime
 
 /**
  * Mapping of a single issue from neo4j to github
+ * @param imsProject IMSProject of the repo
```

Comment:
please be consistent with the way you document properties at least in the same class

Reply:
97e7d2bad00bc5df055901a6122ebc81ba380105